docs/content/guide/component.ngdoc

```diff
@@ -233,9 +233,13 @@ it upwards to the heroList component, which updates the original data.
 </file>
 
 <file name="heroDetail.js">
-  function HeroDetailController($scope, $element, $attrs) {
+  function HeroDetailController($scope) {
     var ctrl = this;
 
+    ctrl.hero = ctrl.hero || {};
+    ctrl.hero.name = ctrl.hero.name || undefined;
+    ctrl.hero.location = ctrl.hero.location || 'unknown';
```

Comment:
TBH, I would rather remove this initialization and the corresponding test.
It doesn't add much value and doesn't seem like a good practice either. If nothing else, such initialization should be happening in an `$onInit` hook ?

WDYT ?